ArduPlane/Parameters.cpp

```diff
@@ -729,6 +736,8 @@ const AP_Param::Info Plane::var_info[] = {
     // @DisplayName: speed used for speed scaling calculations
     // @Description: Airspeed in m/s to use when calculating surface speed scaling. Note that changing this value will affect all PID values
     // @Units: m/s
+    // @Range: 0 250
```

Comment:
Way to large a range for a slider to be useful for most people. It's okay to have people manually set something outside the range on this if they need it. I'd cut this down to something more like 50 m/s max.

Reply:
fixed. i didn't think about the slider size actually...